thefuck/rules/wrong_hyphen_before_subcommand.py

```diff
@@ -0,0 +1,22 @@
+from thefuck.utils import get_all_executables
+from thefuck.specific.sudo import sudo_support
+
+
+@sudo_support
+def match(command):
+    if '-' not in command.script_parts[0] or command.script_parts[0] in get_all_executables():
+        return False
+
+    cmd, subcmd = command.script_parts[0].split('-')[:2]
+    if cmd not in get_all_executables():
```

Comment:
This could simply be :

```python
return cmd in get_all_executables()
```

Reply:
@yotam180, can you please address this remark?
```suggestion
    return cmd in get_all_executables()
```

This along with [the one a few boxes above](https://github.com/nvbn/thefuck/pull/977#discussion_r336793322):
```diff
-    cmd, subcmd = command.script_parts[0].split('-')[:2]
+    cmd = command.script_parts[0].split('-', 1)[0]
```